R/calcEmissions4ReportExtra.R

```diff
@@ -0,0 +1,15 @@
+#' Calculates historical emissions needed for estimating emission factors
+#' in extra emissions reporting
+#
+#'
+#'
+#' @return list of magclass with CEDS CH4 and N2O emissionsdata for
+#' CEDS sectors for timesteps 2000-2100, in Mt N or Mt CH4 per year
```

Comment:
timesteps 2000-2100 -> 2020

Reply:
Good catch. I also had to add a `sectors` parameter now, and redid the docs